test/functional/sqlsrv/0065.phpt

```diff
@@ -91,90 +86,97 @@ BEGIN
     set @param = @param + @param;
 END;
 PROC;
-$s = sqlsrv_query( $c, $create_proc );
-if( $s === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+$s = sqlsrv_query($c, $create_proc);
+if ($s === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-$s = sqlsrv_query( $c, 'SELECT c1, c2, c3 FROM utf8test' );
-if( $s === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+$s = sqlsrv_query($c, 'SELECT c1, c2, c3 FROM utf8test');
+if ($s === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-if( sqlsrv_fetch( $s ) === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+if (sqlsrv_fetch($s) === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-$t = sqlsrv_get_field( $s, 0, SQLSRV_PHPTYPE_STRING( 'utf-8' ));
-if( $t === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+$t = sqlsrv_get_field($s, 0, SQLSRV_PHPTYPE_STRING('utf-8'));
+if ($t === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-if( $t != "So?e sä???? ?SCII-te×t" ) {
-    die( "varchar(100) doesn't match So?e sä???? ?SCII-te×t" );
+if ($t != "So?e sä???? ?SCII-te×t") {
+    die("varchar(100) doesn't match So?e sä???? ?SCII-te×t");
 }
 
-$t = sqlsrv_get_field( $s, 1, SQLSRV_PHPTYPE_STRING( 'utf-8' ));
-if( $t === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+$t = sqlsrv_get_field($s, 1, SQLSRV_PHPTYPE_STRING('utf-8'));
+if ($t === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-if( $t != $utf8 ) {
-    die( "nvarchar(100) doesn't match the inserted UTF-8 text." );
+if ($t != $utf8) {
+    die("nvarchar(100) doesn't match the inserted UTF-8 text.");
 }
 
-$t = sqlsrv_get_field( $s, 2, SQLSRV_PHPTYPE_STRING( 'utf-8' ));
-if( $t === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+$t = sqlsrv_get_field($s, 2, SQLSRV_PHPTYPE_STRING('utf-8'));
+if ($t === false) {
+    die(print_r(sqlsrv_errors(), true));
 }
 
-if( $t != $utf8 ) {
-    die( "nvarchar(max) doesn't match the inserted UTF-8 text." );
+if ($t != $utf8) {
+    die("nvarchar(max) doesn't match the inserted UTF-8 text.");
 }
 
-sqlsrv_free_stmt( $s );
+sqlsrv_free_stmt($s);
 
 // test proc to baseline with
 $t = 1;
+$sqlType = AE\isColEncrypted() ? SQLSRV_SQLTYPE_INT : null;
 
-$s = sqlsrv_query( $c, "{call IntDoubleProc(?)}", array( array( &$t, SQLSRV_PARAM_INOUT, SQLSRV_PHPTYPE_INT )));
+$s = sqlsrv_query($c, "{call IntDoubleProc(?)}", array(array(&$t, SQLSRV_PARAM_INOUT, SQLSRV_PHPTYPE_INT, $sqlType)));
```

Comment:
seems like the test originally tests for SQLSRV_SQLTYPE_INT anyway, so there's no need you for to differentiate the two